src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -128,7 +128,7 @@ def create(call, parent=None):
         # Create the loop over the appropriate entity.
         # Avoid circular import
         # pylint: disable=import-outside-toplevel
-        from psyclone.dynamo0p3 import DynLoop
+        from psyclone.domain.lfric import LFRicLoop
```

Comment:
Can this be moved to the top if it is made more specific: `from psyclone.domain.lfric.lfric_loop import...`?

Reply:
oops, missed this, will tackle after lunch

Reply:
no, LFRicLoop uses LFRicBuiltins and vice versa